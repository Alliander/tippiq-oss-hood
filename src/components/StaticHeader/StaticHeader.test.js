import React from 'react';
import renderer from 'react-test-renderer';
import Element from './index';

jest.mock('../../piwik', () => ({
  push: jest.fn(),
}));

test('Renders', () => {
  const component = renderer.create(
    <Element />
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});

