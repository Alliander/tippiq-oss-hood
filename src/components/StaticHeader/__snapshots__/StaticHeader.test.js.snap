exports[`test Renders 1`] = `
<div
  className="row">
  <div
    className={undefined}>
    <div
      className="undefined undefined">
      <span>
        Meld je aan voor het Buurtbericht en ontdek jouw buurt!
      </span>
    </div>
    <div
      className="undefined undefined">
      <a
        className="btn btn-primary btn-block"
        onClick={[Function]}
        style={Object {}}
        url="/registreren">
        Nu aanmelden
      </a>
    </div>
  </div>
  <div
    className={undefined}>
    <ul
      className={undefined}>
      <li>
        <i
          aria-hidden="true"
          className="fa fa-check undefined" />
         Je kunt het overzicht personaliseren
      </li>
      <li>
        <i
          aria-hidden="true"
          className="fa fa-check undefined" />
         Ontvangt wekelijks het buurbericht
      </li>
      <li>
        <i
          aria-hidden="true"
          className="fa fa-check undefined" />
         Stel notificaties in voor diensten
      </li>
    </ul>
  </div>
</div>
`;
